packages/client/src/clients/objects/index.ts

```diff
@@ -12,6 +12,7 @@ type SetChannelDataInput = {
   objectId: string;
   collection: string;
   channelId: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is the main rule I'm not sure how we want to handle. We use `any` in a number of places, especially as defaults for generics. I think we can use `unknown` as well which will require users to pass in a type to the generic. We could also disable this rule and allow explicit any while still disallowing implicit any.

Reply:
A few thoughts:
- Any places where are using `any`, but what we really mean is any key/value object, it looks like we have `GenericData` we can use?
- Any places where are using `any` but there's more we can type, we probably should eventually. So maybe slap a TODO in those places?
  - Separately, I'm curious how auto SDK generation efforts will come into play here. Because I imagine when we auto generate node sdk, there will be some types generated as well, and I imagine we would want/be able to re-use those? So maybe this is all an temporary issue.

Reply:
That's a really good point. I'll replace all of these with `GenericData` as that's what we want, then we just have a single eslint disable comment.

Reply:
Update: Great suggestion from Thomas to use `GenericData` means we really only have 1 place where have an explicit any